test_plans/sycl_ext_oneapi_composite_device.asciidoc

```diff
@@ -0,0 +1,128 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for sycl_ext_oneapi_composite_device
+
+THis is a test plan for the API described in the 
+https://github.com/intel/llvm/blob/e94b24718e60a7fa03ca1abbde4f7e37bbd0557d/sycl/doc/extensions/proposed/sycl_ext_oneapi_composite_device.asciidoc[`sycl_ext_oneapi_composite_device`]
+extension specification.
+
+== Testing scope
+
+=== Device coverage
+
+All of the tests described below are performed only on the default device that
+is selected on the CTS command line.
+
+=== Feature test macro
+
+All of the tests should use `#ifdef SYCL_EXT_ONEAPI_COMPOSITE_DEVICE` so they
+can be skipped if the extension is not supported.
+
+== Tests
+
+Note that some of the sections below closely follow the extension specification
+document and describe tests with narrower scope, which are aimed to check
+specific functionality. There are sections below which describe bigger test
+cases which are intended to check how different elements of the extension work
+together.
+
+=== Enumerating composite devices
+
+==== Through the new `get_composite_devices()` API
+
+This test should check that freestanding function `get_composite_devices()`:
+- has the correct signature
+- does not throw an exception
+- returns the same result on subsequent calls
```

Comment:
Need probably some spacing to be interpreted as AsciiDoc item list?

Reply:
Indeed, fixed in f01911d3f691d211d876b02fd6afb7c876d33851